src/browser/external/neoPlanner.js

```diff
@@ -179,6 +179,19 @@ neo.queryPlan = function(element) {
       details.push({ className: 'padding' })
     }
 
+    if (operator.PageCacheHits || operator.PageCacheMisses) {
```

Comment:
Aren't `PageCacheHits` and `PageCacheMisses` (if present) allow ways come together in the result?
If one is present but the other isn't how does `formatNumber` bellow handle `undefined`?

Reply:
Yes, they come together.
Those are also zero checks since the result sends `PageCacheHits: 0, PageCacheMisses: 0` and in that case we don't want to show them at all. 
But if one of them aren't 0, we want to show them. 
@pe4cey.